src/PriceCalculatorKata/IProduct.cs

```diff
@@ -0,0 +1,11 @@
+namespace PriceCalculatorKata;
+public interface IProduct
+{
+    public static float Tax { get; set; }
```

Comment:
these are interesting... static values in the interface :3 
seems like these should be configurable.